src/pages/cli-legacy/graphql-transformer/codegen.mdx

```diff
@@ -305,15 +313,17 @@ var subscriptionWatcher: Cancellable?
 
 //In your app code
 do {
-  subscriptionWatcher = try appSyncClient?.subscribe(subscription: OnCreateTodoSubscription(), resultHandler: { (result, transaction, error) in
-    if let result = result {
-      print(result.data!.onCreateTodo!.name + " " + result.data!.onCreateTodo!.description!)
-    } else if let error = error {
-      print(error.localizedDescription)
-    }
-  })
+    subscriptionWatcher = try appSyncClient?.subscribe(
+      subscription: OnCreateTodoSubscription(),
+      resultHandler: { (result, transaction, error) in
+          if let result = result {
+              print(result.data!.onCreateTodo!.name + " " + result.data!.onCreateTodo!.description!)
+          } else if let error = error {
+              print(error.localizedDescription)
+          }
+      })
 } catch {
```

Comment:
These lines look like they're halfway (2 chars) indented.  Also recommend outdenting the final paren on a new line.